taca/utils/bioinfo_tab.py

```diff
@@ -116,17 +130,18 @@ def get_status(run_dir):
 
     return status
 
-
-
-
+#Returns project, FC, lane & sample (sample-run) status for a given folder
 def get_ss_projects(run_dir):
-    pattern=re.compile("(P[0-9]{3,5})_[0-9]{3,5}")
-    project_ids=set()
+    proj_tree = Tree()
+    proj_pattern=re.compile("(P[0-9]{3,5})_[0-9]{3,5}")
```

Comment:
I called that pattern, not proj_pattern. I don't think there was a proj_pattern in the previous version